docs/shp_build_create.md

```diff
@@ -32,13 +29,13 @@ shp build create <name> [flags]
       --retention-succeeded-limit uint           number of succeeded BuildRuns to be kept (default 65535)
       --retention-ttl-after-failed duration      duration to delete a failed BuildRun after completion
       --retention-ttl-after-succeeded duration   duration to delete a succeeded BuildRun after completion
-      --source-bundle-image string               source bundle image location, e.g. ghcr.io/shipwright-io/sample-go/source-bundle:latest
-      --source-bundle-prune pruneOption          source bundle prune option, either Never, or AfterPull (default Never)
+      --source-oci-artifact-image string         source oci artifact image location, e.g. ghcr.io/shipwright-io/sample-go/source-bundle:latest
+      --source-oci-artifact-prune pruneOption    source oci artifact image prune option, either Never, or AfterPull (default Never)
```

Comment:
Nit: I think I should try uppercase since it's an acronym. 
```suggestion
      --source-oci-artifact-image string         source OCI artifact image location, e.g. ghcr.io/shipwright-io/sample-go/source-bundle:latest
      --source-oci-artifact-prune pruneOption    source OCI artifact image prune option, either Never, or AfterPull (default Never)
```

Reply:
Done.